site/_js/slider.js

```diff
@@ -0,0 +1,183 @@
+import {gsap} from 'gsap';
+import {ScrollTrigger} from 'gsap/ScrollTrigger';
+import {ScrollToPlugin} from 'gsap/ScrollToPlugin';
+
+gsap.registerPlugin(ScrollTrigger, ScrollToPlugin);
+
+const cssClasses = {
+  NAVIGATION_WRAPPER: 'navigation-wrapper',
+  FIXED_NAVIGATION_WRAPPER: 'fixed-navigation-wrapper',
+  YEAR_NAVIGATION: 'year-navigation',
+  FIXED_YEAR_NAVIGATION: 'fixed-year-navigation',
+  MENU: 'menu',
+  FIXED_MENU: 'fixed-menu',
+  MENU_ITEM: 'menuitem',
+  MENU_ITEM_LINK: 'sectionLink',
+  CHROME_LOGO: 'chrome-logo',
+  YEAR_SECTION: 'inner-card',
+  HIDE_NAVIGATION: 'hide-navigation',
+};
+
+class Slider {
+  constructor() {
+    this.currentIndex = 0;
+    this.previousIndex = 0;
+
+    this.initialize();
+  }
+
+  initialize() {
```

Comment:
As mentioned elsewhere, this is absolutely not code that we want to run on every page of the site. Executing a `scroll` listener can [negatively impact site performance](https://blog.chromium.org/2016/05/new-apis-to-help-developers-improve.html), and while it might be needed for the Chrome 100 page, it definitely should not run on all our pages.

Reply:
Done.